### PR TITLE
docs(live_stream): Add Live Stream samples and tests

### PR DESCRIPTION
**Pull request by irataxy**
b:241866780

Part 1 of 3: Inputs. See also https://cloud.google.com/livestream/docs/how-to/create-input-endpoints

Live Stream API needs to be enabled in test project

### CODE REVIEW

File: google-cloud-video-live_stream/samples/create_input.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  operation.response
+end
+
+
+if $PROGRAM_NAME == __FILE__
+  args    = ARGV.dup
+  command = args.shift
+
+  case command
+  when "create_input"
```
Comment (NivedhaSenthil) [inline google-cloud-video-live_stream/samples/create_input.rb:55]: we would not need the command to be separate as we have only one method each file.

File: google-cloud-video-live_stream/samples/get_input.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  name = client.input_path project: project_id, location: location, input: input_id
+
+  # Get the input.
+  input = client.get_input name: name
+
+  # Print the input name.
+  puts "Input: #{input.name}"
+  # [END livestream_get_input]
+
+  input
```
Comment (NivedhaSenthil) [inline google-cloud-video-live_stream/samples/get_input.rb:37]: we generally do not return values in samples method, showing how to access them should be good

**Review by NivedhaSenthil — CHANGES_REQUESTED** [review]

**Comment by NivedhaSenthil** [comment]
Please refer https://googlecloudplatform.github.io/samples-style-guide/ for more details on samples guidelines.
Let us if we can be of any help for fixing the sample's test failures

File: google-cloud-video-live_stream/samples/get_input.rb (hunk 2 above)
Comment (irataxy) [reply google-cloud-video-live_stream/samples/get_input.rb:37]: I'm confused. The current guidance says to follow the Secret Manager samples (https://screenshot.googleplex.com/BT4Y87v3VRDtV2e). Secret Manager returns values (see https://github.com/googleapis/google-cloud-ruby/blob/main/google-cloud-secret_manager/samples/snippets.rb). Note that I also returned values in the Transcoder samples (https://github.com/googleapis/google-cloud-ruby/commit/5edc40b36938dc58b9974fe98c6f2f2e0e3b6050). Has the guidance changed?

**Comment by irataxy** [comment]
> 

Please enable the Live Stream API in the test project.

File: google-cloud-video-live_stream/samples/get_input.rb (hunk 2 above)
Comment (NivedhaSenthil) [reply google-cloud-video-live_stream/samples/get_input.rb:37]: yes the sample guidelines have been updated.. please refer this PR for the structure https://github.com/googleapis/google-cloud-ruby/pull/19090/files#diff-c81b6c61172b0db53f3f62f36421248553ba8112b29e32b1f24813446a817312..

File: google-cloud-video-live_stream/samples/create_input.rb (hunk 1 above)
Comment (NivedhaSenthil) [reply google-cloud-video-live_stream/samples/create_input.rb:55]: Please refer this PR https://github.com/googleapis/google-cloud-ruby/pull/19090/files#diff-c81b6c61172b0db53f3f62f36421248553ba8112b29e32b1f24813446a817312 we do not need this CLI

File: google-cloud-video-live_stream/samples/get_input.rb (hunk 2 above)
Comment (irataxy) [reply google-cloud-video-live_stream/samples/get_input.rb]: Refactored

File: google-cloud-video-live_stream/samples/create_input.rb (hunk 1 above)
Comment (irataxy) [reply google-cloud-video-live_stream/samples/create_input.rb]: Refactored

File: google-cloud-video-live_stream/samples/acceptance/helper.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  let :input do
+    operation = client.create_input(
+      parent: input_location_path,
+      input:  {
+        type: Google::Cloud::Video::LiveStream::V1::Input::Type::RTMP_PUSH
+      },
+      input_id: input_id
+    )
+    operation.wait_until_done!
+    input = operation.response
```
Comment (irataxy) [inline google-cloud-video-live_stream/samples/acceptance/helper.rb:42]: @NivedhaSenthil This line fails the lint check but all of the tests pass. Inputs are long-running operations, and I'd like to set :input = to the response of the operation, which is the actual Live Stream input. What can be done here?

Comment (NivedhaSenthil) [reply google-cloud-video-live_stream/samples/acceptance/helper.rb:42]: sorry for the late response.. missed the ping here
we could just leave the response unassigned to a variable, just `operation.response` in the last line should set it to the let :input

Comment (NivedhaSenthil) [inline google-cloud-video-live_stream/samples/acceptance/helper.rb:42]:
```suggestion
    operation.response
```

Comment (irataxy) [reply google-cloud-video-live_stream/samples/acceptance/helper.rb]: done

**Review by NivedhaSenthil — APPROVED** [review]

**Pull request merged**